packages/accounts-password/password_server.js

```diff
@@ -66,7 +66,11 @@ Accounts._checkPassword = function (user, password) {
   password = getPasswordString(password);
 
   if (! bcryptCompare(password, user.services.password.bcrypt)) {
-    result.error = new Meteor.Error(403, "Incorrect password");
+    if(Accounts._options.ambiguousErrorMessages) {
+      result.error = new Meteor.Error(403, "Login failure. Please check your username and password.");  
```

Comment:
Just to add - I updated this in the common/sample `handleError` function I mentioned above.